### PR TITLE
Move large chunk of liquidate() inside unchecked{}

### PR DESCRIPTION
**Pull request by haydenshively**

### CODE REVIEW

File: core/src/Borrower.sol [hunk 1]
```diff
@@ -125,48 +125,40 @@ contract Borrower is IUniswapV3MintCallback {
...
-            uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96) + incentive1 / strain;
-            TOKEN1.safeTransfer(address(callee), maxLoss1);
+                TOKEN1.safeTransfer(address(callee), maxLoss1);
+                callee.callback0(data, maxLoss1, converted0);
 
-            callee.callback0(data, maxLoss1, converted0);
+                repayable0 += converted0;
+            } else {
+                uint256 converted1 = liabilities1 / strain;
+                uint256 maxLoss0 = Math.mulDiv(converted1 + incentive1 / strain, Q96, priceX96);
```
Comment (aneesh1) [inline core/src/Borrower.sol:152]: Subject to potential change per this [discussion](https://github.com/aloelabs/aloe-ii/pull/30#discussion_r1073222041).

File: core/src/Borrower.sol [hunk 2]
```diff
@@ -125,48 +125,40 @@ contract Borrower is IUniswapV3MintCallback {
...
-            // If both are zero or neither is zero, there's nothing more to do
-            // TODO: compensate liquidators for txn costs using ANTE
-        } else if (liabilities0 > 0) {
-            uint256 converted0 = liabilities0 / strain;
+            if (liabilities0 + liabilities1 == 0 || (liabilities0 > 0 && liabilities1 > 0)) {
+                // If both are zero or neither is zero, there's nothing more to do.
+                // Callbacks/swaps won't help.
+            } else if (liabilities0 > 0) {
+                uint256 converted0 = liabilities0 / strain;
+                uint256 maxLoss1 = Math.mulDiv(converted0, priceX96, Q96) + incentive1 / strain;
```
Comment (aneesh1) [inline core/src/Borrower.sol:144]: Subject to potential change per this [discussion](https://github.com/aloelabs/aloe-ii/pull/30#discussion_r1073222041).

**Review by aneesh1 — APPROVED** [review]
Good - this seems like a cut and paste change. To check, I just looked at the diff and made sure the statements were the same. Interested to see the gas savings that result from this change per: https://ethereum.stackexchange.com/a/117359.

File: core/src/Borrower.sol (hunk 1 above)
Comment (haydenshively) [reply core/src/Borrower.sol:152]: I promise we'll come back to this :)

**Pull request merged**